Fix the problem with the URL

octopusinvitro opened the pull request:
This PR fixes the problem with the URL:

```
fatal: Not a git repository (or any of the parent directories): .git
Could not determine git repo for 'scraped_page_archive' to use.

See https://github.com/everypolitician/scraped_page_archive#usage for details.
/app/vendor/bundle/ruby/2.0.0/gems/vcr-3.0.3/lib/vcr/request_handler.rb:97:in `on_unhandled_request':  (VCR::Errors::UnhandledHTTPRequestError)
```

**But the scrapter is still broken**, as there are some cells in the table that contain no picture, for which the scraper gets a `nilclass` error when trying to call methods on that element, since there is no `img` element.

tmtmtmtm commented:
I think this needs a better commit message, to explain what you've actually done (and why). At a casual glance, it looks like the main thing you did was change it from running on Ruby 2.0.0…

octopusinvitro committed:
Fix library version conflicts

    * Removes the dependency of Ruby version by deleting it in the Gemfile since there were dependency conflicts (we don't remember exactly which at this point)
    * Deletes/regenerates the Gemfile.lock so that we use the right versions of the libraries and avoid conflicting version numbers

octopusinvitro committed:
Fix the problem with the URL

    * We added the MORPH_SCRAPER_CACHE_GITHUB_REPO_URL environment variable to morph.io to solve the error:
    Not a git repository (or any of the parent directories): .git
    Could not determine git repo for 'scraped_page_archive' to use.

    * This commit requires the whole scraped_page_archive instead of scraped_page_archive/open-uri subclass to solve the error:
     bad URI(is not URI?): git@github.com:everypolitician-scrapers/bangladesh-parliament-scraper.git (URI::InvalidURIError)

tmtmtmtm commented:
Woot! ❇️  🎆  👍
